architecture/core_concepts/pods_and_services.adoc

```diff
@@ -161,6 +161,49 @@ link:https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod docu
 has details about the functionality and purpose of pods.
 ====
 
+[[admin-manage-pod-preset]]
+=== Injecting Information into Pods Using Pod Presets
+
+A _pod preset_ is an object that injects user-specified information into pods as they are created.
+
+[IMPORTANT]
+====
+Pod presets is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+Using pod preset objects you can inject:
+
+* xref:../../dev_guide/secrets.adoc#dev-guide-secrets[secret objects]
+* xref:../../dev_guide/configmaps.adoc#consuming-configmap-in-pods[`ConfigMap` objects]
+* xref:../../dev_guide/volumes.adoc#dev-guide-volumes[storage volumes]
+* container volume mounts
+* environment variables
+
+Developers only need make sure the pod labels match the label selector on the PodPreset in order to add all that information to the pod. The xref:../../architecture/core_concepts/pods_and_services.adoc#labels[label] on a pod associates the pod with one or more pod preset objects that have a matching xref:../../architecture/core_concepts/pods_and_services.adoc#services[label selectors].
+
+Using pod presets, a developer can provision pods without needing to know the details about the services the pod will consume. An administrator can keep configuration items of a service invisible from a developer without preventing the developer from deploying pods.
```

Comment:
Is there extra spacing before "An administrator"?

Reply:
No.